docs/upgrade-notes.asciidoc

```diff
@@ -48,6 +48,42 @@ For Elastic Security solution release information, refer to {security-guide}/rel
 [float]
 === Breaking changes
 
+[discrete]
+[[breaking-207091]]
+.Removed legacy security rules bulk endpoints (9.0.0)
+[%collapsible]
+====
+*Details* +
+--
+* `POST /api/detection_engine/rules/_bulk_create` has been replaced by `POST /api/detection_engine/rules/_import`
+* `PUT /api/detection_engine/rules/_bulk_update` has been replaced by `POST /api/detection_engine/rules/_bulk_action`
+* `PATCH /api/detection_engine/rules/_bulk_update has been replaced by `POST /api/detection_engine/rules/_bulk_action`
+* `DELETE /api/detection_engine/rules/_bulk_delete` has been replaced by `POST /api/detection_engine/rules/_bulk_action`
+* `POST api/detection_engine/rules/_bulk_delete` has been replaced by `POST /api/detection_engine/rules/_bulk_action`
+--
+These changes were introduced in PR {kibana-pull}#197422[#197422]
```

Comment:
Made a few minor changes:
- Removed the hash symbol as it might be messing up the link. The link leads users to [this page](https://github.com/elastic/kibana/pulls#197422) instead of https://github.com/elastic/kibana/pull/197422. 
- Removed "PR" because users can see they're viewing a PR if they click the link.
- Added a period at the end since this is a complete sentence.  

```suggestion
These changes were introduced in {kibana-pull}197422[#197422].
```